rust_sodium-sys/build.rs

```diff
@@ -54,24 +54,21 @@ use std::io::Cursor;
 ///
 /// If something fails, an error message string is returned.
```

Comment:
That comment is probably now outdated :)

Reply:
Fixed thanks!